src/Compatibility/Maps/src/Android/MapRenderer.cs

```diff
@@ -212,16 +214,16 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 			{
 				if (NativeMap != null)
 				{
-					MoveToRegion(Element.LastMoveToRegion, false);
+				//	MoveToRegion(Element.LastMoveToRegion, false);
```

Comment:
Are these intentionally commented out?

Reply:
Yes we removed this api, but i want to leave the code there if we need to check something on the "old renderer"